docker/Dockerfile

```diff
@@ -1,26 +1,22 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
 
-FROM mcr.microsoft.com/azureml/curated/acpt-pytorch-2.2-cuda12.1:22
+FROM mcr.microsoft.com/azureml/curated/acpt-pytorch-2.2-cuda12.1:30
 
 RUN apt-get update && apt-get install -y \
     git \
     zip \
     vim
 
+# Required by pyvista PIP package
+# TODO must also enable xserver
+RUN apt-get install -y libglfw3-dev libgles2-mesa-dev
+
 WORKDIR /app/
 RUN wget https://developer.download.nvidia.com/devtools/nsight-systems/NsightSystems-linux-cli-public-2023.4.1.97-3355750.deb
 RUN apt install ./NsightSystems-linux-cli-public-2023.4.1.97-3355750.deb
 
-# WARNING
-# mgmetis wrongly specifies Cython a "install time" requirement and mpi4py
-# as optional but they should both be "setup time" requirements.
-# Therefore we need to install them separatedly before
-# `pip install mgmetis` or `pip install -r dev-requirements.txt`
-# so that ParMETIS support can be added.
-RUN pip install \
-	Cython==3.0.11 \
-	mpi4py==3.1.5
+RUN pip uninstall -y torch
 
 COPY dev-requirements.txt /app/dev-requirements.txt
 RUN pip install -r dev-requirements.txt
```

Comment:
I wonder whether this part really matters or not, because we will install EASIER and its dependencies within a conda environment.